.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+  # Enable version updates for Gradle
+  - package-ecosystem: "gradle"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    target-branch: "main"
+    reviewers:
+      - "@DataBiosphere/broadworkspaces"
+      - "@rtitle"
+    labels:
+      - "dependency"
+      - "gradle"
+    commit-message:
+      prefix: "[No Ticket]"
```

Comment:
Yunyi is following examples from other repos-- for instance, https://github.com/DataBiosphere/duos-ui/blob/develop/.github/dependabot.yml.